### PR TITLE
Timeout Warning & Build Instructions 

### PR DESCRIPTION
**Pull request by jlpoolen**
1) prints to console timeout value, if previously set and causing the termination of the process
2) isolated Build instructions with its own heading.  In text mode, a link at the end of an introductory paragraph can be missed.

This is my second pull request, the previous one failed because I had not correctly run checkstyle.py _after_ committing rather than before.

### CODE REVIEW

**Comment by naushir** [comment]
Thank you for your PR.  Happy with the changes themselves, however, would you be able to provide two separate commits:

- One for the build instructions change.
- Second one for the timeout log message.

The merge commits and formatting fixes should not be part of the PR.

File: apps/libcamera_vid.cpp [hunk 1]
```diff
@@ -96,6 +96,9 @@ static void event_loop(LibcameraEncoder &app)
 		bool frameout = options->frames && count >= options->frames;
 		if (timeout || frameout || key == 'x' || key == 'X')
 		{
+		        if (timeout)
+				std::cout << "Halting: reached timeout of " << options->timeout << " milliseconds.\n";
+
 			app.StopCamera(); // stop complains if encoder very slow to close
```
Comment (naushir) [inline apps/libcamera_vid.cpp:102]: This wants to be std::cerr

**Comment by jlpoolen** [comment]
Will do.

File: apps/libcamera_vid.cpp (hunk 1 above)
Comment (jlpoolen) [reply apps/libcamera_vid.cpp:102]: Agreed.

**Comment by jlpoolen** [comment]
Redoing per request to isolate changes and direct to standard error.

**Pull request closed without merging**